views/HtmlStatic.svelte

```diff
@@ -131,7 +131,7 @@
   <div
     id="q-election-seats-svg-container"
     class="q-election-seats-svg-container" />
-  <div class="q-election-seats-total s-font-note">{item.totalSeats} Sitze</div>
+  <div class="q-election-seats-total s-font-note"><span class="s-font-note--tabularnumss-font-note--tabularnums">{item.totalSeats}</span> Sitze</div>
```

Comment:
Twice the same CSS class without a space in between. Copy + paste error?

Reply:
Nice catch, sometimes Flycut has it's hickups!